xonsh/tools.py

```diff
@@ -926,9 +926,11 @@ def SLICE_REG():
 
 
 def ensure_slice(x):
-    """Convert a string or int to a slice."""
-    if x is None:
+    """Convert `x` to a slice. If `x` is already a slice object return as is."""
```

Comment:
I guess numpy doc is using a purposefully undefined part of rst: http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup